src/components/sheet/SimpleSheet.tsx

```diff
@@ -0,0 +1,51 @@
+import { IS_ANDROID } from '@/env';
+import { useDimensions } from '@/hooks';
+import React from 'react';
+import { ScrollView, StatusBar } from 'react-native';
+import { getStatusBarHeight } from 'react-native-iphone-x-helper';
```

Comment:
I'm worried this library is outdated and unmaintained

Reply:
hmm good point i will look into another solution